primefaces/src/test/java/org/primefaces/util/FileUploadUtilsTest.java

```diff
@@ -396,7 +401,7 @@ void requireValidFilename_NotBlank() {
         ValidatorException ex = assertThrows(ValidatorException.class, () -> FileUploadUtils.requireValidFilename(invalidFileName));
 
         // Assert
-        assertEquals("Filename cannot be empty or null", ex.getFacesMessage().getDetail());
+        assertEquals("primefaces.FileValidator.FILENAME_NOT_NULL", ex.getFacesMessage().getDetail());
```

Comment:
shouldnt you check the real message here instead of the key?

Reply:
It can't it blows up because three is not FacesContext. 

```
Unexpected exception type thrown, expected: <javax.faces.validator.ValidatorException> but was: <java.lang.NullPointerException>
        at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
        at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3128)
        at org.primefaces.util.FileUploadUtilsTest.requireValidFilename_NotBlank(FileUploadUtilsTest.java:401)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.NullPointerException: Cannot invoke "javax.faces.context.FacesContext.getApplication()" because "facesContext" is null
        at org.primefaces.util.MessageFactory.getMessage(MessageFactory.java:142)
        at org.primefaces.util.MessageFactory.getMessage(MessageFactory.java:136)
        at org.primefaces.util.FileUploadUtils.requireValidFilename(FileUploadUtils.java:95)
        at org.primefaces.util.FileUploadUtilsTest.lambda$11(FileUploadUtilsTest.java:401)
        at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
```

Reply:
Wait I think I can mock the static testing it now.

Reply:
Had to wrap it in this and it works!

```java
 try (MockedStatic<FacesContext> facesContext = Mockito.mockStatic(FacesContext.class)) {
            facesContext.when(FacesContext::getCurrentInstance).thenReturn(context);

}

```